README.md

```diff
@@ -35,18 +35,32 @@ Add the Filestack File Picker and initialization script to your layout:
 
 ```erb
 <%= filestack_js_include_tag %>
-<%= filestack_js_init_tag %> 
+<%= filestack_js_init_tag %>
 ```
 
-Please note: the scripts need to be added before your application's custom scripts, e.g. before any scripts in your assets folder, if you need access the Filestack client in your own Javascript. 
+Please note: the scripts need to be added before your application's custom scripts, e.g. before any scripts in your assets folder, if you need access the Filestack client in your own Javascript.
 
 Set your API key and client name in `config/application.rb`:
 
 ```ruby
-config.filestack_rails.api_key = "Your Filestack API Key"
-config.filestack_rails.client_name = "custom_client_name"
+config.filestack_rails.api_key = 'Your Filestack API Key'
+config.filestack_rails.client_name = 'custom_client_name'
+```
+The client name defaults to "filestack_client" and is injected into your client-side Javascript. This is because v3 of the File Picker lives in the Javascript of your web application. For more information, please see our [File Picker documenation](https://www.filestack.com/docs/javascript-api/pick-v3).
+
+### CNAME
+
+CNAME is a custom add on feature that can be used along with other customization to create a seamless whitelabeled filepicker for your application.
```

Comment:
I don't think the sales information should live in the GitHub -- just what they need to use CNAME if they have it set up. Could we change this to something like:

"If you have set up a custom CNAME set up, you can add it to your filestack_rails config. The Picker will modify all assets to formatted with your domain origin instead of Filestack's." 

Or something like that. And then we can go into the example code.

Reply:
@staturecrane  thanks for review :) I will rewrite it a little bit different.
What about CHANGELOG file? should I also write about changes with CNAME configuration?

Reply:
@gabifiolek Once the PR is merged into develop, then you can follow our [wiki guidelines](https://github.com/filepicker/wiki/wiki/Releases) for release, which includes updating the CHANGELOG and version with the appropriate info.